servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -124,14 +124,23 @@ void RenderForwardMobile::fill_push_constant_instance_indices(SceneState::Instan
 	p_instance_data->reflection_probes[0] = 0xFFFFFFFF;
 	p_instance_data->reflection_probes[1] = 0xFFFFFFFF;
 
+	RendererRD::ForwardID sorted_reflection_probes[MAX_RDL_CULL];
+	for (uint32_t i = 0; i < p_instance->reflection_probe_count; i++) {
+		sorted_reflection_probes[i] = p_instance->reflection_probes[i];
+	}
+
+	std::sort(sorted_reflection_probes, sorted_reflection_probes + p_instance->reflection_probe_count, [&](RendererRD::ForwardID a, RendererRD::ForwardID b) {
```

Comment:
We don't use std::sort, so this should use the SortArray class instead.

Reply:
Done! I'm still unfamiliar with the SortArray class, I just followed suit with how the probe instances are sorted by size with the `ReflectionProbeInstanceSort` struct in `update_reflection_probe_buffer()`, so I hope that approach is fine here as well. :)